GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.ts

```diff
@@ -188,6 +188,16 @@ namespace gdjs {
       let currentIndex = 0;
       while (currentIndex < path.length - 1) {
         const key = path[currentIndex];
+        console.log(key);
+        const parsedKey = key.toString().match(/(.*)\((.*)\)/);
```

Comment:
Can you explain when this is used? This seems a new addition/feature, could maybe be in another PR, or is it needed here?

Reply:
Before, it would have done something like `variable._children.childname._children.secondChildName`, but now that wouldn't work as there are also arrays. I added those "function calls" to call `getChild`, that automatically finds out if it is an array or structure and adequately returns from _children or _childrenList.

Reply:
See my other comment. I suggest we make the debugger inspector/transformer smarter, so that it can properly use _children or _childrenList according to the type (we know the type, because we show them to the user).

This will allow to remove this regex, and the crash that would happen when I try to modify the value of a variable called `Something (with parenthesis)` :)

Reply:
Note that we could argue that this format (an array of strings, that denote members, until the last one which denote a function to call) is limited (you can't call a function in the middle of it).
This is fine as long as we can do without it. We can always rework it later if we want a new format that would allow each part of the path to be either 1) a property name or 2) a function name with arguments.

It would not be a hard problem, just a matter of reworking what we send from the editor, adapt typing and this code. But it's also a bit more complexity that we don't need, because we can make our variable inspector a bit smarter by checking the type of each variable when constructing the path.